pkg/autoscaler/multiscaler.go

```diff
@@ -48,7 +49,10 @@ type Metric struct {
 
 // MetricSpec is the parameters in which the Revision should scaled.
 type MetricSpec struct {
-	TargetConcurrency float64
+	TargetConcurrency      float64
+	TargetConcurrencyPanic float64
```

Comment:
Should this be called something like `PanicThresholdConcurrency`? IIRC we're not changing the target concurrency in panicmode but we enter it when the concurrency passes a certain threshold.

Reply:
Yeah, I went back and forth on this.  I wanted the X and Y dimensions of the stable window (`window` and `target` respectively) to be associated with the X and Y dimensions of the panic window.  What 4 names would you propose for these dimensions?